### PR TITLE
chore: update rollup among other deps

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

Not sure why Renovate closed the PR.

### CODE REVIEW

File: .yarn/releases/yarn-1.22.4.js [hunk 1]
```diff
@@ -34805,15 +34805,27 @@ function hasMergeConflicts(str) {
 function parse(str, fileLoc) {
   const parser = new Parser(str, fileLoc);
   parser.next();
-  try {
-    return parser.parse();
-  } catch (error1) {
+
```
Comment (tdeekens) [inline .yarn/releases/yarn-1.22.4.js:34808]: Updating yarn.

File: packages/application-components/src/components/modal-pages/internals/modal-page.tsx [hunk 2]
```diff
@@ -29,9 +29,8 @@ import ModalPageTopBar from './modal-page-top-bar';
 const getDefaultParentSelector = () =>
   process.env.NODE_ENV === 'test'
     ? document.body
-    : (document.querySelector<HTMLElement>(
-        `#${PORTALS_CONTAINER_ID}`
-      ) as HTMLElement);
+    : (document.querySelector<HTMLElement>(`#${PORTALS_CONTAINER_ID}`) as
```
Comment (tdeekens) [inline packages/application-components/src/components/modal-pages/internals/modal-page.tsx:32]: Prettier.

File: rollup.config.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ const {
 } = require('@commercetools-frontend/mc-scripts/package.json');
 const resolve = require('rollup-plugin-node-resolve');
 const json = require('@rollup/plugin-json');
-const commonjs = require('rollup-plugin-commonjs');
+const commonjs = require('@rollup/plugin-commonjs');
```
Comment (tdeekens) [inline rollup.config.js:13]: The other plugin is actually deprecated by now.

**Comment by emmenko** [comment]
I can take a look later

**Comment by tdeekens** [comment]
It's really odd. It's not Jest for sure. No clue what can cause this.

**Comment by tdeekens** [comment]
Also not sentry or our sdks. So odd.

**Comment by tdeekens** [comment]
It's really hard to tell what causes this problem. The changelog for apollo is not available. The dependency is pulled in even without updating apollo (due to a ^ range) on their end. Looking at the code changes I also can't tell what the reason is.

**Comment by emmenko** [comment]
So is it working now with the pinned dependency?

**Comment by tdeekens** [comment]
> So is it working now with the pinned dependency?

Yes.

**Pull request merged**